src/CookiePhysicsComponent.cpp

```diff
@@ -60,7 +62,7 @@ void CookiePhysicsComponent::updatePhysics(float deltaTime) {
             cookieState.hits++;
             cookieState.hitPositions.push_back(holder_->getPosition());
             if( objHit->cookieDeliverable) {
-                objHit->changeShader("Green");
+                objHit->changeMaterial(materialManager.getMaterial("Red Rubber"));
```

Comment:
Nice catch, do we want the houses to turn red upon delivery though? Red to me usually means you done goofed lol